.env

```
@@ -0,0 +1,2 @@
+COMPOSE_PROJECT_NAME=azure_ad_delta_sync
```

Align this with the repositiory name:
```suggestion
COMPOSE_PROJECT_NAME=azure-ad-delta-sync
```

---

Removed.